pants

```diff
@@ -362,17 +362,27 @@ function bootstrap_pants {
   local pants_version="$1"
   local python="$2"
   local pants_sha="${3:-}"
+  local pants_debug="${4:-}"
 
   local pants_requirement="pantsbuild.pants==${pants_version}"
   local maybe_find_links
   if [[ -z "${pants_sha}" ]]; then
     maybe_find_links=""
   else
     maybe_find_links="--find-links=$(find_links_url "${pants_version}" "${pants_sha}")"
-   fi
+  fi
+
+  local debug_suffix
+  if [[ -z "${pants_debug}" ]]; then
+    debug_suffix=""
+  else
+    debug_suffix="-debug"
+    pants_requirement="${pants_requirement} debugpy==1.6.0"
```

Comment:
as this var is now used un-quoted, maybe worth a comment to _not_ use constraints like `>` or such ;)

Reply:
Or maybe I use an array.

Reply:
OK switched to arrays